src/Modules/Frontend/Domain/Block/AbstractBlockController.php

```diff
@@ -31,7 +31,7 @@ abstract class AbstractBlockController implements BlockControllerInterface
     protected ?Response $responseOverride = null;
     private string $templatePath;
     /** @var array<string, mixed> */
-    private array $assignedContent = [];
+    protected array $assignedContent = [];
```

Comment:
why did you change this? Not a fan of making this protected

Reply:
I needed a way to pass the id / extra id of the ContentBlock to the widget. I saw this used to be done with $this->data['id'] but not sure how that was passed on from the Page/Block editor, hence the changes here and to the assign methods. Do you have any other ideas or suggestions?

Reply:
The ContentBlock entity that is linked to the widget already has the extra id

Reply:
assignedContent made private again in https://github.com/forkcms/forkcms/pull/3555/commits/c24fac573aad3d9e429b9a352c24087b2dd1f862